src/utility/Albany_Meta_Utils.hpp

```diff
@@ -0,0 +1,226 @@
+#ifndef ALBANY_META_UTILS_HPP
```

Comment:
Since this file was taken verbatim from an open source library, we need to put a header stating it. Something like

```
// This file is copied from "ekat_meta_utils.hpp", a file in the EKAT library (https://github.com/E3SM-Project/EKAT).
```